### PR TITLE
Add bootstrap, no theme

### PR DESCRIPTION
**Pull request by brentm5**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ gem 'rails', '4.0.0.rc1'
 
 gem 'pg'
 gem 'sass-rails', '~> 4.0.0.rc1'
+gem 'high_voltage'
+gem 'bootstrap-sass', '~>2.3.1.0'
```
Comment (drapergeek) [inline Gemfile]: I don't think we need any version specification.

**Comment by drapergeek** [comment]
This should also include a basic application.html.erb.

File: Gemfile (hunk 1 above)
Comment (brentm5) [inline Gemfile]: Just was being cautious because of rails 4

Comment (drapergeek) [inline Gemfile]: Yea I get that but we have  Gemfile.lock, it just isn't necessary.

**Comment by brentm5** [comment]
What do you want for that? nav and hero?

**Comment by drapergeek** [comment]
The log out doesn't work.

**Comment by brentm5** [comment]
Fixed

**Comment by drapergeek** [comment]
:+1: Squash and merge it!

**Pull request closed without merging**